components/apimgt/org.wso2.carbon.apimgt.rest.api.core/src/main/java/org/wso2/carbon/apimgt/rest/api/core/impl/ApisApiServiceImpl.java

```diff
@@ -58,4 +63,39 @@ public Response apisApiIdGatewayConfigGet(String apiId, String accept,
                     MediaType.APPLICATION_JSON).entity(errorDTO).build();
         }
     }
+
+    /**
+     *
```

Comment:
There is an issue with the JavaDoc and I have comment it in https://github.com/wso2/carbon-apimgt/pull/4232#pullrequestreview-47072166
Please have a look.